ocfweb/main/openbugbounty.py

```diff
@@ -0,0 +1,11 @@
+from django.http import HttpResponse
```

Comment:
can we update the naming here (the module and function), maybe to `security_dot_txt` or something instead? `security.txt` isn't an openbugbounty thing.

https://securitytxt.org/